package.json

```diff
@@ -20,7 +20,7 @@
     "hasura:console": "npm run hasura console -- --no-browser",
     "hasura:migrate:init": "npm run hasura migrate create \"init\" -- --from-server",
     "hasura:migrate:apply": "npm run hasura migrate apply",
-    "test": "lerna run test --parallel",
+    "test": "lerna run test --parallel --",
```

Comment:
what's this for ?

Reply:
So we can forward args to the test command in each package for CI / GH Actions (e.g. --ci and --coverage). Without it the command gets consumed by lerna instead of forwarding to the test command.